drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -815,4 +815,11 @@ config NRF_WIFI_PS_INT_PS
 	  whether to stay in PS (for lower amount of buffered data) or exit PS (for higher
 	  amount of buffered data).
 endchoice
+
+config NRF70_SCAN_PASSIVE
```

Comment:
the name should be `NRF70_FORCED_PASSIVE_SCAN` or `NRF70_PASSIVE_SCAN_ONLY` to reflect its function